packages/shared/src/components/fields/form/FormWrapper.tsx

```diff
@@ -50,7 +50,7 @@ export function FormWrapper({
   );
 
   return (
-    <div className={classNames('flex flex-col', className?.container)}>
+    <div className={classNames('flex w-full flex-col', className?.container)}>
```

Comment:
Is this only for mobile, won't it affect desktop forms as well?

Reply:
The FormWrapper is so far only used for mobile renders it has a couple places but for instance:
https://github.com/dailydotdev/apps/blob/22a9bd91b0e24986cdfc7c2647e60e1196f0fea0/packages/shared/src/components/modals/common/Modal.tsx#L136